contrib/win32/install/install/product.wxs

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?if $(var.MANUFACTURER) = "" ?>
+	<?define MANUFACTURER = "Microsoft Corporation"?>
+<?endif?>
+
+<?if $(var.SOURCEDIR) = "" ?>
+  <?error Variable SOURCEDIR not set?>
+<?endif?>
+
+<?if $(var.VERSION) = "" ?>
+  <?define VERSION = "0.0.0.0"?>
+<?endif?>
+
+<?if $(sys.BUILDARCH)="x86" ?>
+  <?define PROGRAMFILESFOLDER="ProgramFilesFolder"?>
+<?elseif $(sys.BUILDARCH)="x64" ?>
+  <?define PROGRAMFILESFOLDER="ProgramFiles64Folder"?>
+<?else?>
+  <?error Unsupported value of sys.BUILDARCH=$(sys.BUILDARCH)?>
+<?endif?>
+
+<?define UPGRADECODERELEASE = "c7a00fb1-c477-40d3-a951-ec4c749da439" ?>
+<?define UPGRADECODEDEBUG = "e2dd3c95-7a8b-444e-b75b-3e1cf697aadc" ?>
+
+<?if $(var.PRODUCTID) = "" ?>
+  <?define PRODUCTID = "*" ?>
```

Comment:
I see you are changing it, so I don't understand the hardcoded product ID in the README above. Whatever the case, this won't work as expected unless the version is incremented with the build.

Reply:
Because I simply want to have the ability to go with "best practices" even if I don't like them 😁